<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>110</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Postitoimipaikka</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Etunimi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addMemberLastNameLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Postinumero</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addMemberAddPushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lisää</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addMemberCityLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sukunimi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>120</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jakeluosoite</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addMemberZIPLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addMemberPostalAddressLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addMemberFirstNameLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addMemberCancelPushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peru</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="addMemberPhoneNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>100</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Puhelinnumero</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>addMemberFirstNameLineEdit</tabstop>
  <tabstop>addMemberLastNameLineEdit</tabstop>
  <tabstop>addMemberPostalAddressLineEdit</tabstop>
  <tabstop>addMemberZIPLineEdit</tabstop>
  <tabstop>addMemberCityLineEdit</tabstop>
  <tabstop>addMemberCancelPushButton</tabstop>
  <tabstop>addMemberAddPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
